.github/workflows/ci_cd_pr.yml

```diff
@@ -140,8 +125,9 @@ jobs:
 
   doc:
     name: "Documentation build"
-    runs-on: ubuntu-latest
+    runs-on: [public-ubuntu-latest-8-cores]
```

Comment:
Do we need to run on 8 cores for the PR check?

Reply:
reverted